### PR TITLE
i180 pwa icon fix

### LINKED ISSUE #180: PWA Icon Fix
App icon is squished (maybe just have the feet instead of the feet and name?) also the app name has no spaces but should
Note: tested on iPhone

### PR DESCRIPTION
**Pull request by YunnTun**
## Change Summary

**[Briefly summarise the changes that you made. Just high-level stuff]**

### Change Form

**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [ ] The pull request title has an issue number
- [ ] The change works by "Smoke testing" or quick testing
- [ ] The change has tests
- [ ] The change has documentation

## Other Information

**[Is there anything in particular in the review that I should be aware of?]**

# Related Issue

- Resolve #180

### CODE REVIEW

File: client/vite.config.ts [hunk 1]
```diff
@@ -24,48 +24,68 @@ export default defineConfig({
...
-          // },
-          // {
-          //   src: "icon-512.png",
-          //   sizes: "512x512",
-          //   type: "image/png",
-          //   purpose: "any maskable",
-          // },
-          // {
-          //   src: "public/images/CSF_Logo_WHITE.png",
-          //   sizes: "512x512",
```
Comment (ChantelleKerr) [inline client/vite.config.ts:51]: remove comments

**Review by ChantelleKerr — CHANGES_REQUESTED** [review]
Remove ReloadPWA template

**Review by RadioArsenic — APPROVED** [review]
Good work!

**Review by ChantelleKerr — APPROVED** [review]
nice

**Pull request merged**